src/battle-scene.ts

```diff
@@ -1250,9 +1252,16 @@ export default class BattleScene extends SceneBase {
     this.waveCountText.setVisible(true);
   }
 
-  updateMoneyText(): void {
-    this.moneyText.setText(`₽${Utils.formatLargeNumber(this.money, 1000)}`);
-    this.moneyText.setVisible(true);
+  updateMoneyText(forceVisible: boolean = true): void {
+    if (this.money === undefined) {
+      return;
+    }
+    const formattedMoney =
+			this.moneyFormat === MoneyFormat.ABBREVIATED ? Utils.formatLargeNumber(this.money, 1000) : this.money.toLocaleString("en-US");
```

Comment:
@CodeTappert 
> While we are at it. Couldnt we make the "en-US" also language specific?

Sure, I wanted to do that in the first place, but I saw that the locale was explicitly set here, while the [default behavior is to use the user locale](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#using_tolocalestring). I don't know if this was set for a specific reason.

If this was just by convention, I believe it's ok to use the user locale.
```suggestion
			this.moneyFormat === MoneyFormat.ABBREVIATED ? Utils.formatLargeNumber(this.money, 1000) : this.money.toLocaleString();
```